jsx/Form.js

```diff
@@ -411,6 +412,7 @@ var DateElement = React.createClass({
       label: '',
       value: '',
       id: null,
+      maxYear: '9999-12-31',
```

Comment:
Should not it be something like now() or currentDate()? This would also solve the DoB set to "Tomorrow" or "Next year" at the same time.

Reply:
ohh and also, i believe the max attribute in the input element of type=date takes in a string, not a Date() instance